### PR TITLE
OPENEUROPA-582: Update images on docker-compose and drone file.

### PR DESCRIPTION
**Pull request by voidtek**

### CODE REVIEW

**Comment by pfrenssen** [comment]
This build is failing.

File: composer.json [hunk 1]
```diff
@@ -66,6 +67,7 @@
     },
     "config": {
         "sort-packages": true,
+        "preferred-install": "source",
```
Comment (ademarco) [inline composer.json]: @voidtek any reason why we need to specify `"preferred-install": "source"`?

Comment (ademarco) [inline composer.json]: @voidtek any reason why we need to specify `"preferred-install": "source"`?

File: composer.json [hunk 2]
```diff
@@ -54,7 +54,8 @@
         "composer-exit-on-patch-failure": true,
         "patches": {
             "drupal/core": {
-                "https://www.drupal.org/project/drupal/issues/2943172": "https://www.drupal.org/files/issues/2018-07-05/2943172-kernel-test-base-3.patch"
+                "https://www.drupal.org/project/drupal/issues/2943172": "https://www.drupal.org/files/issues/2018-07-05/2943172-kernel-test-base-3.patch",
+                "https://www.drupal.org/project/drupal/issues/2599228": "https://www.drupal.org/files/issues/2018-05-17/2599228-51.patch"
```
Comment (ademarco) [inline composer.json]: We can remove this patch since it get it in from the multilingual.

Comment (voidtek) [reply composer.json]: I removed the patch.

**Pull request merged**